src/main/java/org/veupathdb/service/eda/ds/plugin/standalonemap/FloatingScatterplotPlugin.java

```diff
@@ -101,7 +102,7 @@ protected void writeResults(OutputStream out, Map<String, InputStream> dataStrea
     varMap.put("overlay", spec.getOverlayVariable());
     String valueSpec = spec.getValueSpec().getValue();
     String yVarType = util.getVariableType(spec.getYAxisVariable());
-    String overlayValues = listToRVector(spec.getOverlayValues());
+    String overlayValues = listToRVector(spec.getOverlayValues().stream().map(BinRange::getBinLabel).collect(Collectors.toList()));
```

Comment:
so right now the R supporting this doesnt know about continuous overlays, but that is going to have to change very soon. something for me to think about mostly i guess...

Reply:
Ah yeah, I just made the minimal possible changes to the floating plugins to get everything compiling for now